### PR TITLE
A new package: legion

### PR DESCRIPTION
**Pull request by junghans**

### CODE REVIEW

File: var/spack/repos/builtin/packages/legion/package.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            return 'Debug'
+        else:
+            return 'Release'
+
+    def cmake_args(self):
+        return ['-DLegion_USE_GASNet=ON', '-DLegion_BUILD_EXAMPLES=ON']
+
+    def build(self, spec, prefix):
+        with working_dir(self.build_directory):
+            make()
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/legion/package.py]: This is the default `build()` implementation so you don't need to override it.

File: var/spack/repos/builtin/packages/legion/package.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        spec = self.spec
+        if '+debug' in spec:
+            return 'Debug'
+        else:
+            return 'Release'
+
+    def cmake_args(self):
+        return ['-DLegion_USE_GASNet=ON', '-DLegion_BUILD_EXAMPLES=ON']
+
+    def build(self, spec, prefix):
```
Comment (davydden) [inline var/spack/repos/builtin/packages/legion/package.py]: do you need that, i think default in `CMakePackage` should work just fine.

File: var/spack/repos/builtin/packages/legion/package.py (hunk 1 above)
Comment (alalazo) [inline var/spack/repos/builtin/packages/legion/package.py]: Why is this method necessary? It seems to me it does the same thing as the default one. Am I missing something?

**Comment by adamjstewart** [comment]
Ha, I beat everyone to it.

**Comment by junghans** [comment]
Copy-paste-error!

**Pull request merged**